tests/integration_tests/__init__.py

```diff
@@ -79,8 +72,38 @@ def setUp(self):
         self.addCleanup(self.driver.quit)
 
         self.driver.root_uri = self.get_server_url()
-
-        super(TestUI, self).setUp()
+        self.driver.implicitly_wait(30)
+        self.verificationErrors = []
+        self.wait = WebDriverWait(self.driver, 60)
```

Comment:
I don't think this will do anything unless used by an explicit wait (ie [Expected Condition](https://selenium-python.readthedocs.io/waits.html))

Can you remove it if that's the case?

Reply:
I don't know how that wait got in there!

Reply:
Actually, this tells selenium to wait at most 30 seconds and try again if it doesn't see something it expects

Reply:
I think the above line (setting up the implicit wait, `self.driver.implicitly_wait(30)`) does that.

As far as I know, explicit waits must be invoked on individual elements (see below example), and I don't see any references to the `self.wait` property in any of the added test functions.

---
Explicit wait example from docs ([see *Expected Conditions* section](https://selenium-python.readthedocs.io/waits.html#explicit-waits)):
```
from selenium.webdriver.support import expected_conditions as EC

wait = WebDriverWait(driver, 10)
element = wait.until(EC.element_to_be_clickable((By.ID, 'someid')))
```